### PR TITLE
Fix duplicate chat connection id crash

### PR DESCRIPTION
**Pull request by MarcusLongmuir**
Fixes an issue where old/flaky connections that a client has abandoned are still tracked by the server and retry attempts from the client would result in a collision and an error was thrown.

The change in behaviour is to explicitly reject these new incoming attempts in favour of maintaining the old connection until that old connection fails a heartbeat.

**What kind of change does your PR introduce?** (check at least one)

- [x] Bugfix

### CODE REVIEW

File: packages/3d-web-experience-client/src/Networked3dWebExperienceClient.ts [hunk 1]
```diff
@@ -360,7 +360,7 @@ export class Networked3dWebExperienceClient {
       this.networkChat = new ChatNetworkingClient({
         url: this.config.chatNetworkAddress,
         sessionToken: this.config.sessionToken,
-        websocketFactory: (url: string) => new WebSocket(`${url}?id=${this.clientId}`),
```
Comment (MarcusLongmuir) [inline packages/3d-web-experience-client/src/Networked3dWebExperienceClient.ts:363]: Drive-by cleanup

**Review by TheCodeTherapy — APPROVED** [review]
